errors.go

```diff
@@ -9,6 +9,12 @@ import (
 )
 
 var (
+	// ErrComputeSystemDoesNotExist is an error returned when the compute system being operated on no longer exists
+	ErrComputeSystemDoesNotExist = syscall.Errno(0xc037010e)
+
+	// ErrElementNotFound is an error returned when the object being referenced does not exist
+	ErrElementNotFound = syscall.Errno(0x490)
```

Comment:
I guess I don't understand what this code is doing. Why are we treating "known errors" differently from other error codes? How are we decided what is "known" and what is not?